### PR TITLE
Combined PR of KaleoFormsProcessSubmissionsDetails testcases: LRQA-70700, LRQA-70701, LRQA-70702, LRQA-70703, LRQA-70704

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LRQA-70703

### CODE REVIEW

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/KaleoFormsNewSubmissions.macro [hunk 1]
```diff
@@ -78,6 +167,20 @@ definition {
...
+			key_status = "${lbstatus}",
+			locator1 = "KaleoFormsNewSubmissions#SUBMISSION_STATUS_LABEL");
+	}
+
+	macro viewVersionOnDetails {
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
+
+		AssertElementPresent(
+			locator1 = "KaleoFormsAdmin#WORKFLOW_VERSION",
+			value1 = "1.0");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/KaleoFormsNewSubmissions.macro:181]: Please, pass "value1" as a variable, cause the workflow version value can be changed. Thanks!

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/KaleoFormsNewSubmissions.macro [hunk 2]
```diff
@@ -78,6 +167,20 @@ definition {
...
 	}
 
+	macro viewStatusLabel {
+		AssertElementPresent(
+			key_status = "${lbstatus}",
+			locator1 = "KaleoFormsNewSubmissions#SUBMISSION_STATUS_LABEL");
+	}
+
+	macro viewVersionOnDetails {
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/macros/KaleoFormsNewSubmissions.macro:177]: Create a macro to do this and use directily on the testcase file please

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 3]
```diff
@@ -73,30 +73,95 @@ definition {
...
+			ddlLastName = "Userln",
+			ddlnotes = "Notes",
+			ddlPhoneMobile = "(123) 9999-9999",
+			ddlPhoneOffice = "(123) 1111-2222",
+			ddlUserName = "testliferay");
 
-		// TODO LRQA-70703 CanViewProcessStatusPending pending implementation
+		KaleoFormsAdmin.saveProcessRecord();
 
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:162]: Use the macro created to this

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 4]
```diff
@@ -73,30 +73,95 @@ definition {
...
+			ddlCompany = "ライフレイ",
+			ddlEmail = "ライフレイ@liferay.com",
+			ddlFirstName = "ライフレイ");
+
+		KaleoFormsAdmin.saveProcessRecord();
+
+		Alert.viewSuccessMessage();
 
-		// TODO LRQA-70702 CanInsertEspecialCaracter pending implementation
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:89]: Use the macro created to this

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 5]
```diff
@@ -109,21 +174,60 @@ definition {
...
+			ddlJobTitle = "QA Engineer",
+			ddlLastName = "Userln",
+			ddlnotes = "Notes",
+			ddlPhoneMobile = "(123) 9999-9999",
+			ddlPhoneOffice = "(123) 1111-2222",
+			ddlUserName = "testliferay");
+
+		KaleoFormsAdmin.saveProcessRecord();
+
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:195]: Use the macro created to this

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 6]
```diff
@@ -73,30 +73,95 @@ definition {
...
+			ddlLastName = "Userln",
+			ddlnotes = "Notes",
+			ddlPhoneMobile = "(123) 9999-9999",
+			ddlPhoneOffice = "(123) 1111-2222",
+			ddlUserName = "testliferay");
+
+		KaleoFormsAdmin.saveProcessRecord();
 
-		// TODO LRQA-70704 CanViewProcessStatusApproved pending implementation
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:116]: Use the macro created to this

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 7]
```diff
@@ -73,30 +73,95 @@ definition {
...
+			workflowAssetType = "Process",
+			workflowTask = "Review");
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Kaleo Forms Admin");
+
+		KaleoFormsAdmin.viewProcessRecord(kfProcessName = "Kaleo Forms Process");
+
+		LexiconEntry.gotoEllipsisMenuItem(menuItem = "View");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:138]: Use the macro created to this

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase [hunk 8]
```diff
@@ -109,21 +174,60 @@ definition {
...
+			ddlJobTitle = "QA Engineer",
+			ddlLastName = "Userln",
+			ddlnotes = "Notes",
+			ddlPhoneMobile = "(123) 9999-9999",
+			ddlPhoneOffice = "(123) 1111-2222",
+			ddlUserName = "testliferay");
+
+		KaleoFormsAdmin.saveProcessRecord();
+
+		KaleoFormsNewSubmissions.viewVersionOnDetails();
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsProcessSubmissionsDetails.testcase:230]: Remeber to pass the variable here after you change the macro

**Review by lucasperj — CHANGES_REQUESTED** [review]

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

**Review by lucasperj — APPROVED** [review]

**Comment by lucasperj** [comment]
ci:forward

**Pull request closed without merging**